admin-dev/themes/new-theme/js/components/grid/extension/action/row/customer/delete-customer-row-action-extension.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+const $ = window.$;
+
+/**
+ * Class DeleteCustomerRowActionExtension handles submitting of row action
+ */
+export default class DeleteCustomerRowActionExtension {
+
+  constructor() {
+    return {
+      extend: (grid) => this.extend(grid),
+    };
+  }
+
+  /**
+   * Extend grid
+   *
+   * @param {Grid} grid
+   */
+  extend(grid) {
+    grid.getContainer().on('click', '.js-delete-customer-row-action', (event) => {
+      event.preventDefault();
+
+      const $deleteCategoriesModal = $('#' + grid.getId() + '_grid_delete_customers_modal');
```

Comment:
sure, i usually forget to use it. :/

but if you take a look at this
```javascript
$('#' + grid.getId() + '_grid_delete_customers_modal')
```
and this
```javascript
$(`#${grid.getId()}_grid_delete_customers_modal`)
```

for me its much easier to read concatenation thats written in old syntax, especially when theres more than one variable.

Reply:
Mmmm I have applied ES6 syntax because I saw it was applied on new JS theme code 🤔 

Maybe @PierreRambaud can share his opinion as he asks for it a lot :) ?